routes/api.php

```diff
@@ -17,11 +17,26 @@
     return $request->user();
 });
 
-Route::namespace('Api')->prefix('reciters')->group(function () {
-    Route::delete('/{reciter}', 'RecitersController@destroy');
-    Route::put('/{reciter}', 'RecitersController@update');
-    Route::patch('/{reciter}', 'RecitersController@update');
-    Route::get('/{reciter}', 'RecitersController@show');
-    Route::post('/', 'RecitersController@store');
-    Route::get('/', 'RecitersController@index');
+Route::namespace('Api')->group(function () {
+    Route::prefix('/reciters')->group(function () {
+        Route::prefix('/{reciter}')->group(function () {
+            //album routes
+            Route::prefix('/albums')->group(function () {
+                Route::post('/{album/update', 'AlbumsController@update');
+                Route::delete('/{album}/destroy', 'AlbumsController@destroy');
+                Route::get('/{album}', 'AlbumsController@show');
+                Route::post('/add', 'AlbumsController@store');
+                Route::get('/', 'AlbumsController@index');
+            });
+            //end album routes
+            //{reciter} routes
+            Route::delete('/', 'RecitersController@destroy');
+            Route::put('/', 'RecitersController@update');
+            Route::patch('/', 'RecitersController@update');
+            Route::get('/', 'RecitersController@show');
+            //end {reciter} routes
+        });
+        Route::post('/', 'RecitersController@store');
+        Route::get('/', 'RecitersController@index');
+    });
```

Comment:
This routes list is getting really confusing to look at. 

Can we clean this up by either using `Route::resource()` or not doing prefix groups for each thing? It helps to have routes grouped by concern, rather than URL structure.

```php
// Reciter Routes
Route::prefix('reciters')->group(function () {
	Route::get('/', 'RecitersController@index');
	Route::post('/', 'RecitersController@store');
	Route::get('/{reciter}', 'RecitersController@show');
	Route::put('/{reciter}', 'RecitersController@update');
	Route::patch('/{reciter}', 'RecitersController@update');
	Route::delete('/{reciter}', 'RecitersController@destroy');
});

// Album Routes
Route::prefix('reciters/{reciter}/albums')->group(function () {
	Route::get('/', 'AlbumsController@index');
	Route::post('/', 'AlbumsController@store');
	Route::get('/{album}', 'AlbumsController@show');
	Route::put('/{album}', 'AlbumsController@update');
	Route::patch('/{album}', 'AlbumsController@update');
	Route::delete('/{album}', 'AlbumsController@destroy');
});
```

The above is much easier to read and look through.